### ISSUE TITLE
error when running run.py

### ISSUE THREAD

**Issue by SuperPi911** [odwdinc/DWIN_T5UIC1_LCD#16]
i am installing this on mainsail os and after i did everything on the page i got stuck on run.py

terminal log 

DWIN handshake 
DWIN OK.
http://127.0.0.1:80
Waiting for connect to /tmp/klippy_uds

Unable to connect socket /tmp/klippy_uds [2,ENOENT]

cananyone help me?

**Comment by Hates1209** [comment]
I have the same issue. Did you use the professional firmware before with the other Display firmware?

**Comment by SuperPi911** [comment]
Yes , you just need to modify the files from GitHub , because you got an error when running it , but I already modified it and it is on my GitHub page , just clone my files on your raspberry pi

**Comment by SuperPi911** [comment]
Here's the link for you https://github.com/SuperPi911/DWIN_T5UIC1_LCD

**Comment by SuperPi911** [comment]
Does it work for you?

**Comment by Hates1209** [comment]
I will try it tomorrow! Thanks a lot <3

**Comment by Hates1209** [comment]
No that’s what I try to use the same GitHub..:(

**Comment by SuperPi911** [comment]
No, just delete what you downloaded from here and use the instructions from my page

**Comment by Hates1209** [comment]
Ok I try it.

**Comment by SuperPi911** [comment]
Did you try and it didn't work

**Comment by Hates1209** [comment]
It’s working fine thanks a lot!

**Comment by Lukaspor** [comment]
I had the same initial issue, using your files helped get past the script errors, however after the initial loading bar with the creality logo, it just freezes there afterwards with no loading bar. I previously had the Jyers screen files flashed, is this causing the issue? Do I need to first reflash the original screen firmware?

**Comment by marine1988** [comment]
Mine is not working any idea?

**Comment by Sofronio** [comment]
> i am installing this on mainsail os and after i did everything on the page i got stuck on run.py
> 
> terminal log
> 
> DWIN handshake DWIN OK. http://127.0.0.1:80 Waiting for connect to /tmp/klippy_uds
> 
> Unable to connect socket /tmp/klippy_uds [2,ENOENT]
> 
> cananyone help me?

In line 256 of printerInterface.py, change that to this.
self.ks = KlippySocket('/home/pi/printer_data/comms/klippy.sock', callback=self.klippy_callback)
